cs.html

```diff
@@ -284,7 +301,67 @@ <h2>Methods</h2>
                 <em>Return type:</em> <code><a>QuicSendStream</a></code>
               </div>
             </dd>
-
+            <dt><dfn><code>sendDatagram</code></dfn></dt>
+            <dd>
+              <p>Sends a datagram as defined by [[QUIC-DATAGRAM]].</p>
+              <p>When <code>sendDatagram</code> is called, the user agent
+              <em class="rfc2119" title="MUST">MUST</em> run the following
+              steps:</p>
+              <ol>
+	            	<li>Let <var>data</var> be the first argument.</li>
+                <li>
+                  <p>Let <var>transport</var> be
+                  the <code><a>RTCQuicTransport</a></code> on
+                  which <code>sendDatagram</code> is invoked.</p>
+                </li>
+                <li>
+                  <p>If <code><var>transport</var>'s state</code> is
+                  not <code>connected</code> <a>throw</a>
+                  an <code>InvalidStateError</code> and abort these steps.</p>
+                </li>
+                <li>
+                  <p>If <code><var>data</var> is too large to fit into a
+                  datagram, <a>throw</a> an <code>InvalidArgumentError</code>
+                  and abort these steps.</p>
+                </li>
+                <li>
+                  <p>Let <var>result</var> be a newly created
+                  <code><a>QuicSendDatagramResult</a></code> object
+                  with <code>result.blocked<code> set to false,
+		              <code>result.acked</code> set to a new promise,
+		              and <code>result.unblocked</code> set to a null.</p>
+                </li>
+                <li>
+                  <p>If <var>transport</var> is unable to send the datagram due
+                  to congestion control, set <code>result.blocked</code> to
+                  true, resolve <code>result.acked</code> to false,
+                  set <code>result.unblocked</code> to a new promise,
+                  return <var>result</var> and abort these steps.</p>
+                </li>
+                <li>
+                  <p>Set <code>result.blocked</code> to false,
```

Comment:
It's already set to  false above, I think you can remove:
"Set <code>result.blocked</code> to false"

Reply:
Good point.  I removed it.